apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersion.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Bigtable.V2
+{
+    /// <summary>
+    /// A version which uniquely identifies a cell within a column.
+    /// </summary>
+    public struct BigtableVersion : IComparable<BigtableVersion>, IEquatable<BigtableVersion>
+    {
+        /// <summary>
+        /// Creates a new <see cref="BigtableVersion"/> version from a value.
+        /// </summary>
+        /// <param name="value">
+        /// The version value non-negative version value, or -1 to initialize from the microseconds of DateTime.UtcNow.
```

Comment:
-1 should get a method like `ServerTime()`

Reply:
In another doc we discussed removing the concept of "server time" and always setting the timestamp, so `ServerTime()` is not the right name but I'm not sure what is. Maybe something with `Default` or `Now`.

Reply:
I think `Default` would make more sense than `Now` here since it sounds like we want to avoid naming which implies the versions are always timestamps. I could make this a static method which always returns the version with the timestamp micros for UTC now. (I want to avoid making it a property so users don't assume it would return the same value each time, although `DateTime.Now` is a property...)

So it would look like `var version = BigtableVersion.Default();` That seem ok?